src/main/java/com/netcracker/study/general/Game.java

```diff
@@ -1,13 +1,87 @@
 package com.netcracker.study.general;
 
+import com.netcracker.study.objects.entities.Bullet;
+import com.netcracker.study.objects.entities.Player;
+
 import java.io.FileNotFoundException;
+import java.util.Set;
 
 public class Game {
     private GameField gameField;
+    private int numberOfPlayers;
+    private int currentPlayer;
+    private Phase currentPhase;
 
     public Game(String mapFilePath) throws FileNotFoundException {
         MapReader mapReader = new MapReader(mapFilePath);
         this.gameField = mapReader.createGameFieldFromTextFile();
+        this.numberOfPlayers = 0;
+    }
+
+    public void addPlayer(int positionX, int positionY, String name) {
+        gameField.addPlayer(positionX, positionY, name);
+        this.numberOfPlayers++;
+    }
+
+    public void start(){
+        this.newRound();
+    }
+
+    private void endTurn(){
+        if (currentPlayer ==  numberOfPlayers) {
+            newRound();
+        } else {
+            currentPlayer++;
+        }
+    }
+
+    private void newRound(){
+        currentPlayer = 1;
+        currentPhase = Phase.PRE_MOVE;
+    }
+
+    String playerShoots(Player player, Direction direction) throws IllegalStateException {
```

Comment:
playerShoots should not return String text. I expect it to return killed player IDs and only then text message should be generated